### DISCUSSION TITLE
Welcome to ng-matero Discussions!

### DISCUSSION THREAD

**Discussion by nzbin** [ng-matero/ng-matero#78] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by someshjagtap** [comment]
I would like to say thank you so much for your great and wonderful contribution to building this theme or library publically.
It is really helpful for understanding the navigation, routing, Authority, and widgets which you have provided pretty coolly.
We have used this theme in most of our projects, 
and we feel very comfortable with this but still good to have the following things in library -
1. Inline Edit for Table Grid.
2. Text Overlay for Table Grid Cell
3. hid and show Columns based on Authority and Permissions.

**Reply by nzbin** [reply]
I'm very glad you like this project. For the extensions library, you can submit issues here.
https://github.com/ng-matero/extensions/issues

**Comment by harislap** [comment]
Whenever i clone this project and try to use it and learn from it or use some components , i can't run it even tho i reinstall the node, and angular , it doesn't show me specific errors , but it just doesn't work , any assistance? i'm still new to angular and node

**Reply by ismailix** [reply]
1-) npm i
2-) npm run hmr

**Comment by abhi-2k** [comment]
Is there any way to make seconds(SS) available using ng matereo mat datetimepicker along with hrs(HH) and minutes(MM) in Time AM/PM? Does it already exist ?
